drivers/sensor/qdec_nrfx/qdec_nrfx.c

```diff
@@ -0,0 +1,306 @@
+/*
+ * Copyright (c) 2018, Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <sensor.h>
+#include <spinlock.h>
+
+#include <nrfx_qdec.h>
+#include <hal/nrf_gpio.h>
+
+#define SYS_LOG_DOMAIN "qdec_nrfx"
+#define SYS_LOG_LEVEL CONFIG_SYS_LOG_QDEC_NRFX_LEVEL
```

Comment:
Definition of this Kconfig option seems to be missing.
And perhaps it would be better to use the new logger (_<logging/log.h>_) right away.

Reply:
fixed
as for the new logger, to be honest I would like to leave it as is and commit it faster.
Other drivers are not using it yet so it should be no problem I guess since somebody will have to do a bulk change anyway.